talisker/gunicorn.py

```diff
@@ -219,7 +219,7 @@ def load_config(self):
                 extra={'errorlog': self.cfg.errorlog})
             self.cfg.set('errorlog', '-')
 
-        if self.cfg.loglevel.lower() == 'debug':
+        if self.cfg.loglevel.lower() == 'debug' and self._devel:
```

Comment:
I probably don't have full context here ... but is there a reason to block "debug" level logs just because it's not a devel system?